.codecov.yml

```diff
@@ -38,6 +38,6 @@ coverage:
       typing:
         flags:
         - MyPy
-        target: 70%  # 100%
+        target: 68.5%  # 100%
```

Comment:
@bdraco why did you lower this? The currently reported number is 87.50% per https://app.codecov.io/gh/aio-libs/propcache?flags%5B0%5D=MyPy. Honestly, I'd prefer MyPy coverage to be fixed rather than discarded. Looking into https://app.codecov.io/gh/aio-libs/propcache/blob/master/tests%2Ftest_init.py?flags%5B0%5D=MyPy, most of it is not that hard to address.

Reply:
Do you have a suggestion on how to type `propcache_module`?

Reply:
I guess `ModuleType` will work here https://github.com/aio-libs/propcache/blob/f7ac8b1920f71b0ece2d144733e338c1d1ce8065/tests/conftest.py#L36

Reply:
Doors closing, will do more if wifi works in flight

Reply:
https://github.com/aio-libs/propcache/pull/33

Reply:
https://github.com/aio-libs/propcache/pull/34

Reply:
Any idea why its missing on these lines?

<img width="701" alt="Screenshot 2024-10-06 at 6 16 32 PM" src="https://github.com/user-attachments/assets/6b0881a9-c3c0-47dd-b735-ad85052d8d40">

Reply:
@bdraco try using the `from .api import cached_property as cached_property` syntax in `__init__.py`. I've been trying to figure out similar things recently, and the experience needs improving: https://discuss.python.org/t/is-there-any-tool-that-can-report-type-coverage-of-a-project/34962/4.
It might be a rule we disable, but it still influences the coverage precision metric. So it wouldn't show up in the console output, but type coverage would be incomplete. I also saw mentions of when something in the type resolves to `Any`, that would make the line imprecise/uncovered. And MyPy does not have a `# pragma: no cover` to make it go away.